src/com/xilinx/rapidwright/timing/TimingModel.java

```diff
@@ -1473,7 +1440,7 @@ private float checkForSitePinDelay(List<TimingGroup> groups) {
                 result.add(ts);
             }
         }
-        return result.toArray(new TimingGroup[result.size()]);
+        return result.toArray(new TimingGroup[0]);

```

Comment:
Let's have a static final empty array we can return here.

Reply:
This one's a weird one though. IntellIJ suggests that I don't put the size in there anymore, which is backed by:
https://stackoverflow.com/questions/174093/toarraynew-myclass0-or-toarraynew-myclassmylist-size
But it's unclear whether a static is necessary since it's just syntatic sugar to tell `toArray()` what object type to return.